tools/Makefile

```diff
@@ -78,6 +78,13 @@ chainer_ctc.done: espnet.done
 	. venv/bin/activate && ./install_chainer_ctc.sh
 	touch chainer_ctc.done
 
+kenlm.done: espnet.done
+	rm -rf kenlm
+	git clone https://github.com/kpu/kenlm.git
```

Comment:
Create install_kenlm.sh as same as the other instead of writing it in the Makefile.